site/js/util/api.js

```diff
@@ -49,5 +49,14 @@ module.exports = {
     })
   },
 
+  postRaw(state, path, body = '') {
+    if (!body) return Promise.reject(new Error('Body not provided'))
```

Comment:
Does it make sense to provide a default value of `''` when that default value won't actually work? I'm thinking no default value at all.

Reply:
Oh yeah.